Source/GTMSessionFetcherCore.xcodeproj/project.pbxproj

```diff
@@ -1313,7 +1321,7 @@
 		4FEE142C1BB22328002ACBF8 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "";
```

Comment:
I'm not sure we want this, won't it prevent it from being signed in someone uses the project in their build?

Reply:
I've returned those values. PTAL